spec/integration/rails/async_queue_adapter_spec.rb

```diff
@@ -0,0 +1,18 @@
+require_relative '../rails_helper'
+
+describe "Rails Async Queue Adapter Test", if: RAILS_PRESENT, type: :request do
+  include ActiveJob::TestHelper if RAILS_PRESENT
+  load_rails_hooks(self)
+
+  it "reports exceptions" do
+    Honeybadger.flush do
+      perform_enqueued_jobs do
+        post "/enqueue_error_job"
```

Comment:
Is there a reason to have this run through a controller? Could it just enqueue the job directly?

```suggestion
        ErrorJob.perform_later({some: "data"})
```

Reply:
@joshuap Honestly, probably a combination of initial insecurity and maybe looking for examples in the wrong place. Looking at the code now, I'm not really sure, why I did that. I'll overhaul the test.